src/pages/debrief/debrief.ts

```diff
@@ -30,4 +30,12 @@ export class DebriefPage extends BasePageComponent {
     this.store$.dispatch(new DebriefViewDidEnter());
   }
 
+  endDebrief(): void {
+    if (this.passed) {
+      this.navController.push('PassFinalisationPage');
```

Comment:
You can avoid the else branch if you `return` from here.

Reply:
Fixed that and added screenshot